cmd/argocd/commands/version_test.go

```diff
@@ -0,0 +1,23 @@
+package commands
+
+import (
+	"bytes"
+	"testing"
+
+	argocdclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
+)
+
+func TestShortVersion(t *testing.T) {
+	buf := new(bytes.Buffer)
+	cmd := NewVersionCmd(&argocdclient.ClientOptions{})
+	cmd.SetOutput(buf)
+	cmd.SetArgs([]string{"version", "--short", "--client"})
+	err := cmd.Execute()
+	if err != nil {
+		t.Fatal("Failed to execute short version command")
+	}
+	output := buf.String()
+	if string(output) != "argocd: v99.99.99+unknown\n" {
+		t.Fatalf("expected \"%s\" got \"%s\"", "argocd: v99.99.99+unknown\n", output)
+	}
```

Comment:
Can you please use `assert` package from `github.com/stretchr/testify/assert` and use `assert.Equal()` for the test?

Reply:
@jannfis I've updated